src/spirit-view/spirit-view.html

```diff
@@ -96,6 +96,12 @@
                     }
                 }
                 return true;
+            },
+
+            _removeName: function() {
```

Comment:
The demos show some examples of validation too, and we can do something similar here, binding the next buttons' enabled status to the validation state of the paper-input. There's an attribute shown in the demos too that says that it cannot validate unless there is text in it, probably something else we can use.